L/libjulia/build_tarballs.jl

```diff
@@ -0,0 +1,3 @@
+include("common.jl")
+build_julia(ARGS, v"1.6.0"; jllversion=v"1.7.0")
+build_julia(ARGS, v"1.7.0-beta3"; jllversion=v"1.7.0")
```

Comment:
Are the "jllversion"s listed here indicating the version of the final JLL? So they must always be equal, right? Even/esp. when we add more Julia builds here in the future? In that case, I would put it into a single variable and use that as argument (DRY)

Reply:
Yes, these are always the same, implemented the change.